frontend/src/reducks/store/store.js

```diff
@@ -1,22 +1,26 @@
-import { applyMiddleware, combineReducers, configureStore } from 'redux';
-import { composeWithDevTools } from 'redux-devtools-extension';
-import { logger } from 'redux-logger';
-import thunk from 'redux-thunk';
+import {
+  applyMiddleware,
+  combineReducers,
+  legacycreateStore as configureStore,
+} from "redux";
```

Comment:
We are using single quotations for other files, not double quotations. If you just formatted in your default settings, please just revert this change from your pull request.

Reply:
Sure sit I will fix them.